locales/en-US/learn.ftl

```diff
@@ -61,3 +61,77 @@ learn-nomicon-button = Read the ’nomicon
 learn-unstable-alt = A hand sharing sparkles
 learn-unstable = The Unstable Book has documentation for unstable features that you can only use with nightly Rust.
 learn-unstable-button = Read the unstable book
+
+# learn/get-started.hbs
```

Comment:
These kinds of comments are seen by translators. I would suggest using two pound signs for FTL file section labels. See https://projectfluent.org/fluent/guide/comments.html